webpack/components/extensions/Hosts/ActionsBar/index.js

```diff
@@ -40,45 +59,51 @@ const HostActionsBar = () => {
   }
 
   return (
-    <>
-      <DropdownItem
-        ouiaId="change-content-s-dropdown-item"
-        key="change-content-source-dropdown-item"
-        href={href}
-        isDisabled={selectedCount === 0}
-      >
-        {__('Change content source')}
-      </DropdownItem>
-      <DropdownItem
-        ouiaId="bulk-change-cv-dropdown-item"
-        key="bulk-change-cv-dropdown-item"
-        onClick={openBulkChangeCVModal}
-        isDisabled={selectedCount === 0}
-        isAriaDisabled={!orgId}
-        tooltip={!orgId && __('To change content view environments, a specific organization must be selected from the organization context.')}
-      >
-        {__('Change content view environments')}
-      </DropdownItem>
-      <DropdownItem
-        ouiaId="bulk-packages-wizard-dropdown-item"
-        key="bulk-packages-wizard-dropdown-item"
-        onClick={openBulkPackagesWizardModal}
-        isDisabled={selectedCount === 0}
-        isAriaDisabled={!orgId}
-        tooltip={!orgId && __('To manage host packages, a specific organization must be selected from the organization context.')}
-      >
-        {__('Manage packages')}
-      </DropdownItem>
-      <DropdownItem
-        ouiaId="bulk-errata-wizard-dropdown-item"
-        key="bulk-errata-wizard-dropdown-item"
-        onClick={openBulkErrataWizardModal}
-        isDisabled={selectedCount === 0}
-      >
-        {__('Manage errata')}
-      </DropdownItem>
-
-    </>
+    <MenuItem
+      itemId="content-flyout-item"
+      key="content-flyout"
+      isDisabled={selectedCount === 0}
+      flyoutMenu={(
+        <Menu ouiaId="content-flyout-menu" onSelect={() => setMenuOpen(false)}>
+          <MenuItem
+            itemId="bulk-packages-wizard-dropdown-item"
```

Comment:
under the Menu should be `<MenuContent> <MenuList>` wrappers to avoid the console error

Reply:
awesome, it works!!